pkgs/flutter/default.nix

```diff
@@ -1,14 +1,11 @@
-{ pkgs }:
+{ flutterPackages }:
 
-let
-  mkFlutter = opts: pkgs.callPackage (import ./flutter.nix opts) {};
-in
-mkFlutter rec {
+flutterPackages.mkFlutter rec {
```

Comment:
I think this will only work on unstable right now :sadpanda:.

Reply:
I thought this was added on 20.09, isn't ?

Reply:
Is 20.09 already released :thinking: ?

Reply:
OMG IT IS!!!!!!!!!!!!

Reply:
HAHAHHAHAHA

Reply:
Can we merge this PR next week? I should upgrade my system on this weekend, so I can test your changes too.